src/aoc2020/day2_copy.clj

```diff
@@ -0,0 +1,127 @@
+(ns aoc2020.day2-copy
+  (:require [aoc2020.util :refer [file->seq]]
+            [clojure.string :as str]))
+
+(defn parse-policy-password-str
+  [policy-password-str]
+  (let [[policy letter pwd] (str/split policy-password-str #" ")
+        [arg1 arg2] (->> (str/split policy #"-")
+                         (mapv parse-long))
+        letter (first letter)]
+    {:policy {:arg1   arg1
+              :arg2   arg2
```

Comment:
Is arg a good name for this?

Reply:
> Is arg a good name for this?

It should be applied to both `lo, hi` and `pos1, pos2`, so I used the `arg`.

Reply:
> That doesn't answer the question.

`arg` is not a good name, how about `limit`?